recipes/webtest/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "WebTest" %}
+{% set version = "2.0.33" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41348efe4323a647a239c31cde84e5e440d726ca4f449859264e538d39037fd0
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - beautifulsoup4
```

Comment:
Are these requirements all needed for building the package?

Reply:
Oops. No. Pushing a fix.